src/utils.rs

```diff
@@ -86,7 +86,7 @@ pub fn map_keb_error_code_to_sspi_error(krb_error_code: u32) -> (ErrorKind, Stri
             ErrorKind::TooManyPrincipals,
             "Multiple principal entries database".into(),
         ),
-        KDC_ERR_NULL_KEY => (ErrorKind::EncryptFailure, "The client or server has a null key".into()),
+        KDC_ERR_NULL_KEY => (ErrorKind::EncryptFailure, "The client or server has null key".into()),
```

Comment:
note: if you want to fix the style here, the sentence should not be capitalized either
```suggestion
        KDC_ERR_NULL_KEY => (ErrorKind::EncryptFailure, "the client or server has null key".into()),
```

Reply:
done: https://github.com/Devolutions/sspi-rs/pull/250/commits/b234249ec7c1643afefb50db942816e02754ef62